text_prop.value is of type unsigned char*
The original intention was to check if the pointer is non-null and this is now upstream:
https://github.com/fluxbox/fluxbox/blob/c7ff3696d256c45d27ed3ad37d67821f6293916b/util/fluxbox-remote.cc#L76

diff -rupN fluxbox-1.3.7_orig/util/fluxbox-remote.cc fluxbox-1.3.7/util/fluxbox-remote.cc
--- fluxbox-1.3.7_orig/util/fluxbox-remote.cc	2015-02-08 11:44:45.377187009 +0100
+++ fluxbox-1.3.7/util/fluxbox-remote.cc	2025-10-15 14:24:09.414014138 +0200
@@ -73,7 +73,7 @@ int main(int argc, char **argv) {
     if (strcmp(cmd, "result") == 0) {
         XTextProperty text_prop;
         if (XGetTextProperty(disp, root, &text_prop, atom_result) != 0
-            && text_prop.value > 0
+            && text_prop.value != 0
             && text_prop.nitems > 0) {
 
             printf("%s", text_prop.value);
